pgjdbc/src/main/java/org/postgresql/util/StreamWrapperCleaningAction.java

```diff
@@ -37,7 +40,10 @@ public InputStream getStream() throws IOException {
   }
 
   @Override
-  public void close() throws IOException {
+  public void onClean(boolean leak) throws IOException {
+    if (leak ) {
+      LOGGER.log(Level.WARNING, GT.tr("Finalizing a StreamWrapper that was never closed:"));
+    }
```

Comment:
I'm not sure this is helpful logging.
We should either remove it or add details so the log messages are actionable.

Reply:
Fair point. I do think we should have a way of logging leaks.